deps/jerry/jerry-core/ecma/operations/ecma-exceptions.c

```diff
@@ -134,6 +135,32 @@ ecma_new_standard_error (ecma_standard_error_t error_type) /**< native error typ
 
   ((ecma_extended_object_t *) new_error_obj_p)->u.class_prop.class_id = LIT_MAGIC_STRING_ERROR_UL;
 
+  const char *frames_id_p = "_frames";
+
+  ecma_string_t *frames_str_p = ecma_new_ecma_string_from_utf8 ((const lit_utf8_byte_t *) frames_id_p, 7);
+
+  ecma_property_value_t *prop_value_p = ecma_create_named_data_property (new_error_obj_p,
+                                                                         frames_str_p,
+                                                                         ECMA_PROPERTY_CONFIGURABLE_WRITABLE,
+                                                                         NULL);
+  ecma_deref_ecma_string (frames_str_p);
+
+  uint32_t depth = 10;
+  // create frames
+  uint32_t *frames = malloc(sizeof(uint32_t) * depth);
+  memset(frames, 0, sizeof(uint32_t) * depth);
+  jerry_get_stacktrace_depth(frames, depth);
```

Comment:
`jerry_get_stacktrace_depth()` is defined at [here](https://github.com/Rokid/ShadowNode/blob/e6ced3744bf3bcc8d44ef90edaafebb545b5012c/deps/jerry/jerry-core/api/jerry.c#L3392), and I think we could move the implementation into `jcontext`, and in jerry-core/api/jerry.c, we just call the internal impl as the following:

```c
void
jerry_get_stacktrace_depth (uint32_t *frames, uint32_t depth)
{
  jerry_assert_api_available ();
  return JERRY_BACKTRACE_DEPTH(frames, depth);
}
```

Reply:
BTW, the stacktrace are not regular naming rule in JerryScript, let's use `backtrace` to keep consistent with the community :)